crates/viewer/re_view_spatial/src/pinhole.rs

```diff
@@ -118,17 +116,15 @@ pub fn query_pinhole_and_view_coordinates_from_store_without_blueprint(
         .component_mono_quiet::<components::Resolution>(
             archetypes::Pinhole::descriptor_resolution().component,
         )
-        .unwrap_or_else(|| {
-            resolution_of_image_at(ctx, query, entity_path).unwrap_or([100.0, 100.0].into())
-        });
+        .unwrap_or_default();
```

Comment:
this and below should use fallbacks rather than default otherwise we may get accidental divergence?
That's a problem we have in a bunch of places I believe :/

Reply:
> That's a problem we have in a bunch of places I believe :/

These other places, are those the transforms?